worker/uniter/runner/debug/server.go

```diff
@@ -218,7 +223,11 @@ const debugHooksHookScript = `#!/bin/bash
 echo $$ > $JUJU_DEBUG/hook.pid
 if [ -z "$JUJU_DEBUG_AT" ] ; then
 	exec /bin/bash --noprofile --init-file $JUJU_DEBUG/init.sh
+elif [ ! -x "__JUJU_HOOK_RUNNER__" ] ; then
+	juju-log --log-level INFO "debugging is enabled, but no handler for $JUJU_HOOK_NAME, skipping"
```

Comment:
We could get rid of this logging.
It was needed for me to actually understand what is going on, but it is plausible that it ends up too verbose in actual debug output. Especially since this is a belt-and-suspenders sort of thing, as we shouldn't actually set up hook.sh if we notice there is no script to run. (At one point I was doing it differently.)